pkg/reconciler/apis/permissionclaimlabel/permissionclaimlabel_reconcile.go

```diff
@@ -293,24 +311,20 @@ func (c *controller) getInformerForGroupResource(group, resource string) (inform
 	return nil, schema.GroupVersionResource{}, fmt.Errorf("unable to find informer for %s.%s", group, resource)
 }
 
-func (c *controller) patchGenericObject(ctx context.Context, old, new metav1.Object, gvr schema.GroupVersionResource, lc logicalcluster.Name) error {
-	oldJSON, err := json.Marshal(old)
-	if err != nil {
-		return err
-	}
-	newJSON, err := json.Marshal(new)
+func (c *controller) patchGenericObject(ctx context.Context, obj metav1.Object, gvr schema.GroupVersionResource, lc logicalcluster.Name) error {
+	objJSON, err := json.Marshal(obj)
 	if err != nil {
 		return err
 	}
-	patchBytes, err := jsonpatch.CreateMergePatch(oldJSON, newJSON)
+	patchBytes, err := jsonpatch.CreateMergePatch(objJSON, objJSON)
```

Comment:
? What should this mean? `{}` is the same.

Reply:
done and update

Reply:
not seeing the update. Not pushed yet?